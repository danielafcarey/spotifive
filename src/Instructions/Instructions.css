.Instructions {
  padding-top: 30px; }
  .Instructions .instruction {
    display: flex;
    flex-direction: row;
    align-items: center;
    padding: 0 20px; }
    .Instructions .instruction p {
      padding: 0 15px; }
