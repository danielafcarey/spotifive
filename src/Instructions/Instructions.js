import React from 'react';

const Instructions = () => {
  const search = require('../images/search.png');
  const click = require('../images/click.png');
  const listen = require('../images/listen.png'); 

  return (
    <div className='Instructions'>
      <div className='instruction'>
        <img src={ search } alt='Search'/>
        <p>Search for an artist</p>
      </div>
      <div className='instruction'>
        <img src={ click } alt='Click'/>
        <p>
          Click TryIt to add that artists&apos; top 5 songs 
          to your Spotifive playlist
        </p>
      </div>
      <div className='instruction'>
        <img src={ listen } alt='Listen'/>
        <p>
          See new music in your Spotifive playlist in the Spotify app
        </p>
      </div>
    </div>
  );
};

export default Instructions;
