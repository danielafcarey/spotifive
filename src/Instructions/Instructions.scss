.Instructions {
  padding-top: 30px;

  .instruction {
    display: flex;
    flex-direction: row;
    align-items: center;
    padding: 0 20px;

    p {
      padding: 0 15px;
    }
  }
}
