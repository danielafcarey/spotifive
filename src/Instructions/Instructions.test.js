import React from 'react';
import { shallow } from 'enzyme';
import Instructions from './Instructions';

describe('Instructions', () => {

  it('matches the snapshot', () => {
    const wrapper = shallow(<Instructions />);

    expect(wrapper).toMatchSnapshot();
  })
})
