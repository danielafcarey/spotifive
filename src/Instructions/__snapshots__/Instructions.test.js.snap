// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Instructions />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="instruction"
>
          <img
                    alt="Search"
                    src="search.png"
          />
          <p>
                    Search for an artist
          </p>
</div>,
        <div
          className="instruction"
>
          <img
                    alt="Click"
                    src="click.png"
          />
          <p>
                    Click TryIt to add that artists' top 5 songs to your Spotifive playlist
          </p>
</div>,
        <div
          className="instruction"
>
          <img
                    alt="Listen"
                    src="listen.png"
          />
          <p>
                    See new music in your Spotifive playlist in the Spotify app
          </p>
</div>,
      ],
      "className": "Instructions",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="Search"
              src="search.png"
/>,
            <p>
              Search for an artist
</p>,
          ],
          "className": "instruction",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Search",
              "src": "search.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Search for an artist",
            },
            "ref": null,
            "rendered": "Search for an artist",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="Click"
              src="click.png"
/>,
            <p>
              Click TryIt to add that artists' top 5 songs to your Spotifive playlist
</p>,
          ],
          "className": "instruction",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Click",
              "src": "click.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Click TryIt to add that artists' top 5 songs to your Spotifive playlist",
            },
            "ref": null,
            "rendered": "Click TryIt to add that artists' top 5 songs to your Spotifive playlist",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="Listen"
              src="listen.png"
/>,
            <p>
              See new music in your Spotifive playlist in the Spotify app
</p>,
          ],
          "className": "instruction",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Listen",
              "src": "listen.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "See new music in your Spotifive playlist in the Spotify app",
            },
            "ref": null,
            "rendered": "See new music in your Spotifive playlist in the Spotify app",
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="instruction"
>
            <img
                        alt="Search"
                        src="search.png"
            />
            <p>
                        Search for an artist
            </p>
</div>,
          <div
            className="instruction"
>
            <img
                        alt="Click"
                        src="click.png"
            />
            <p>
                        Click TryIt to add that artists' top 5 songs to your Spotifive playlist
            </p>
</div>,
          <div
            className="instruction"
>
            <img
                        alt="Listen"
                        src="listen.png"
            />
            <p>
                        See new music in your Spotifive playlist in the Spotify app
            </p>
</div>,
        ],
        "className": "Instructions",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="Search"
                src="search.png"
/>,
              <p>
                Search for an artist
</p>,
            ],
            "className": "instruction",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Search",
                "src": "search.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Search for an artist",
              },
              "ref": null,
              "rendered": "Search for an artist",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="Click"
                src="click.png"
/>,
              <p>
                Click TryIt to add that artists' top 5 songs to your Spotifive playlist
</p>,
            ],
            "className": "instruction",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Click",
                "src": "click.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Click TryIt to add that artists' top 5 songs to your Spotifive playlist",
              },
              "ref": null,
              "rendered": "Click TryIt to add that artists' top 5 songs to your Spotifive playlist",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="Listen"
                src="listen.png"
/>,
              <p>
                See new music in your Spotifive playlist in the Spotify app
</p>,
            ],
            "className": "instruction",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Listen",
                "src": "listen.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "See new music in your Spotifive playlist in the Spotify app",
              },
              "ref": null,
              "rendered": "See new music in your Spotifive playlist in the Spotify app",
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
